src/Lucid/EagerLoad/index.js

```diff
@@ -60,10 +60,10 @@ class EagerLoad {
    * @private
    */
   _chainNested ({ relatedQuery }, nested) {
-    if (nested) {
-      const name = _.first(_.keys(nested))
-      relatedQuery.with(name, nested[name])
-    }
+    Object.entries(nested || {})
```

Comment:
Also try using `_.each` here, just that we use lodash everywhere

Reply:
Was wondering about this - do we need it? I mean, most use cases of lodash can be done easily using native API.

Reply:
Yeah but once project is using it, there is no point is mixing both. So until lodash stays, I prefer keeping it uniformed

Reply:
I don't agree with you on that, yet I'll update PR to your request :)

Reply:
Fixed in 3676b2e